app/actions/remote/category.ts

```diff
@@ -13,7 +13,7 @@ export type CategoriesRequest = {
      error?: unknown;
  }
 
-export const fetchCategories = async (serverUrl: string, teamId: string, fetchOnly = false): Promise<CategoriesRequest> => {
+export const fetchCategories = async (serverUrl: string, teamId: string, fetchOnly = false, prune = false): Promise<CategoriesRequest> => {
```

Comment:
probably `prune` should not be the last parameter so that the action complies with the format of the other ones

Reply:
https://github.com/mattermost/mattermost-mobile/pull/5954#discussion_r806748661 ... which way do we want it? I had it the other way around at first.

Reply:
Definitely prune and lastly fetchOnly..